src/templates/genericContentPage.js

```diff
@@ -57,7 +57,16 @@ const GenericContentPage = ({
   },
 }) => (
   <>
-    <Helmet title={title} />
+    <Helmet
```

Comment:
Does this impact the `/about-us` pages at all? If not, could it be split into a separate PR and linked to the correct Jira ticket? If the ticket is too restricting (as this affects multiple pages), feel free to refine it to better reflect what needs done.

Reply:
it affects the moments and rainbow- plates pages because they are generated this way, line 41 and 43. Ya I was going to rename this one to link to the overarching ticket 6 because they are all quite interwoven but got distracted with the eve ticket